src/app/presentation/menu-shortcut/menu-shortcut.component.ts

```diff
@@ -15,8 +15,14 @@ export class MenuShortcutComponent implements OnInit {
 
   ngOnInit() {
   }
+
   onDisplay() {
-    this.onOpenMenu = (this.onOpenMenu === false) ? true : false;
+    this.onOpenMenu = !this.onOpenMenu;
+    document.addEventListener('click', (event) => {
```

Comment:
This adds a global event listener everytime onDisplay is called, right? I don't think that is desired. 

Can you use angular style `(click)="handleMenuButtonClick()"`? If not, please leave a detailed comment here and in the other eventListener you are creating to explain why.